README.adoc

```diff
@@ -117,20 +117,41 @@ method List {
 	// Order criteria.
 	in Order String
 
+        // Kind of the result - "ClusterList".
+        out Kind String
```

Comment:
This is not correct. The "kind" parameter is built-in and shouldn't appear in the specification.

Reply:
OK, dropping from here.  I'll document it's automatically included.